sphinx/source/docs/dev_guide/env_vars.rst

```diff
@@ -34,10 +34,14 @@ This is a meta variable equivalent to the following environment variables:
 - ``BOKEH_PRETTY=true``
 - ``BOKEH_PY_LOG_LEVEL=debug``
 - ``BOKEH_RESOURCES=absolute-dev``
-- ``BOKEH_SIMPLE_IDS=true``
 
 Accepted values are ``yes``/``no``, ``true``/``false`` or ``0``/``1``.
 
+.. note::
+    When running server examples, the ``BOKEH_RESOURCES`` setting that
+    ``BOKEH_DEV`` sets will cause the page to stop rendering. So you 
+    will need to manually also set ``BOKEH_RESOURCES=server``.
```

Comment:
Given this is a dev guid, this should be `server-dev`.

Reply:
`server-dev` just sets to use unminified resources, which`BOKEH_DEV` already does? All things being otherwise equal, less typing is better

Reply:
Nice catch - thanks @mattpap

Reply:
I overlooked BOKEH_MINIFIED, though without looking at the code I'm not sure what takes precedence. In fact I intended to fix this for a while (#3529).

Reply:
Additionally, I think notebook also needs some help to work width BOKEH_DEV (setting BOKEH_RESOURCES=inline I suppose).

Reply:
`BOKEH_DEV=true BOKEH_RESOURCES=server bokeh serve --show sliders.py` uses unminified resources, which is also what I would expect.